en/peripherals/t265_vio.md

```diff
@@ -0,0 +1,31 @@
+# T265 VIO Camera Setup Guide
+
+The [Intel Realsense Tracking Camera T265](https://www.intelrealsense.com/tracking-camera-t265/) is a [VIO](https://dev.px4.io/master/en/advanced/computer_vision.html) (or V-SLAM) camera produced by Intel.
+It provides odometry information which can be used to augment or replace other positioning systems on PX4.
+
+## Setup Instructions
+
+These instructions explain how to set up VIO using ROS (the supported method):
+
+- On the companion computer, install and configure [MAVROS](https://dev.px4.io/master/en/ros/mavros_installation.html) and verify it is connected successfully to the flight controller
+- Get the [VIO bridge ROS node from Auterion](https://github.com/Auterion/VIO_bridge)
+- Configure the VIO bridge launch file to reflect your camera orientation.
```

Comment:
How? That bridge has no information at all on usage in its readme

Reply:
I'll add something there. However the default is "lenses pointing downwards". People shouldn't need to do any configuration if they mount the T265 this way, which from our testing was the most robust.